### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by AntoinouM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +133,33 @@ To test the application, run the following command:
 npm run test
 ```
 
+
+## Contributing
+
+Contributing
```
Comment (esitsolf) [inline README.md:139]: this one is to much

File: README.md [hunk 2]
```diff
@@ -44,6 +133,33 @@ To test the application, run the following command:
...
+- Write or adapt tests as needed.
+- Add or change the documentation as needed.
+- Squash your commits into a single commit with git's [interactive rebase](https://help.github.com/articles/interactive-rebase). Create a new branch if necessary.
+- Push your branch to your fork on Github, the remote ```origin```.
+- From your fork open a pull request in the correct branch. Target the project's ```develop``` branch if there is one, else go for ```master```!
+- If the maintainer requests further changes just push them to your branch. The PR will be updated automatically.
+- Once the pull request is approved and merged you can pull the changes from ```upstream``` to your local repo and delete your extra branch(es).
+
+And last but not least: Always write your commit messages in the present tense. Your commit message should describe what the commit, when applied, does to the code – not what you did to the code.
+
```
Comment (esitsolf) [inline README.md:162]: We don't have a develop and master but dev and main

**Review by esitsolf — CHANGES_REQUESTED** [review]
Good job!
Small changes to do.
I wonder if presenting the PR process is a good thing: who will review them?
Wouldn't it be better to let know that the project is done from our side ?
Maybe we should discuss that with Victor.

File: README.md (hunk 1 above)
Comment (AntoinouM) [reply README.md:139]: What do you mean?

File: README.md (hunk 2 above)
Comment (AntoinouM) [reply README.md:162]: my bad!

File: README.md [hunk 3]
```diff
@@ -44,6 +133,32 @@ To test the application, run the following command:
 npm run test
 ```
 
+
+## Contributing
```
Comment (Vanessi-K) [inline README.md:137]: If we keep this I think it should be the last think before the Licence.

**Review by Vanessi-K — CHANGES_REQUESTED** [review]
I think we at least should remove the comment on how to run the tests, since they are failing.

I think all the `.DS_Stor`e files should not be in version control.

File: README.md (hunk 3 above)
Comment (AntoinouM) [reply README.md]: done

Comment (Vanessi-K) [reply README.md]: I still see them 
image

**Comment by Vanessi-K** [comment]
Could you please adjust a typo in point 6 of the how to update the catalog.
There is w W missing for a with

File: README.md (hunk 3 above)
Comment (AntoinouM) [reply README.md]: should be good!

**Review by Vanessi-K — APPROVED** [review]

**Review by esitsolf — APPROVED** [review]

**Pull request merged**